dts/arm/nuvoton/npcx/npcx7-alts-map.dtsi

```diff
@@ -12,6 +12,7 @@
 		/* SCFG DEVALT 0 */
 		alt0_spip_sl:      alt00     { alts = <&scfg 0x00 0x0 0>; };
 		alt0_gpio_no_spip: alt03_inv { alts = <&scfg 0x00 0x3 1>; };
+		alt0_gpio_no_fpip: alt07_inv { alts = <&scfg 0x00 0x7 1>; };
```

Comment:
This bit determines the pin-mux of flash interface unit (FIU). The four pins (PIN96.A0.A2.A4) of this module connect with the external flash if it is 0. Otherwise, they connect with internal flash instead. In case the user confuses the direction of FIU which ec contains internal flash, our datasheet describes it as a reserved bit which must be 1. In our HAL, its definition is FPIP (FIU SPI Interface Peripheral). If you think reserved7 is better and prevent the user to misuse it, I will update it.

Reply:
If npcx7 (name of this file) never has an external flash, then I would make it reserved7, but it it is possible to have an external flash, then naming it is okay. It might be worth adding a comment though since the name no_fpip isn't in the datasheet (and the other names are in the datasheet).

Reply:
Done.